generators/entity-server/templates/src/main/java/package/service/mapper/_EntityMapper.java

```diff
@@ -34,19 +34,27 @@
           existingMappings.push(relationships[idx].otherEntityNameCapitalized);
       } } } %><%= existingMappings.map(otherEntityNameCapitalized => otherEntityNameCapitalized + 'Mapper.class').join(', ') %>})
 public interface <%= entityClass %>Mapper extends EntityMapper<<%= entityClass %>DTO, <%= entityClass %>> {
+
 <%_
 // entity -> DTO mapping
 var renMapAnotEnt = false; //Render Mapping Annotation during Entity to DTO conversion?
 for (idx in relationships) {
     const relationshipType = relationships[idx].relationshipType;
     const relationshipName = relationships[idx].relationshipName;
     const ownerSide = relationships[idx].ownerSide;
-    if (relationshipType === 'many-to-one' || (relationshipType === 'one-to-one' && ownerSide === true)) {renMapAnotEnt = true;
+    if (relationshipType === 'many-to-one' || (relationshipType === 'one-to-one' && ownerSide === true)) {
+        renMapAnotEnt = true;
 _%>
+    @Mapping(source = "<%= relationshipName %>.id", target = "<%= relationships[idx].relationshipFieldName %>Id")
+<%      if (relationships[idx].otherEntityFieldCapitalized !='Id' && relationships[idx].otherEntityFieldCapitalized !== '') { _%>
```

Comment:
Moreover, it should be `relationships[idx].otherEntityFieldCapitalized !== 'Id' && ...`

Reply:
Only formatting changed, not code. Don't know.

Reply:
What I mean is:
  - `!=` should be `!==` as we strongly prefer strict in/equality checks over the regular ones
  - There should be a space right after the inequality check (formatting)

Reply:
I didn't notice you didn't introduce the code (sorry, the green background got me). However, improving the code each time someone changes it is a good practice. It'd be cool if you could do it.